ui/components/app/assets/asset-list/asset-list.js

```diff
@@ -77,11 +77,17 @@ const AssetList = ({ onClickAsset, showTokensLinks }) => {
   const {
     currency: primaryCurrency,
     numberOfDecimals: primaryNumberOfDecimals,
-  } = useUserPreferencedCurrency(PRIMARY, { ethNumberOfDecimals: 4 });
+  } = useUserPreferencedCurrency(PRIMARY, {
+    ethNumberOfDecimals: 4,
+    withCheckShowNativeToken: true,
```

Comment:
This my be a preference, but I usually like to save the `with` prefix as something for higher order components, rather than boolean flags.

Don't need to make any change here, but just caught my eye.

Reply:
hmmm, Would it be better to rename to `shouldCheckShowNativeToken`

Reply:
yeah I like the sound of that better. You can make the call though if you think it's worth it 👍

Reply:
Done here [1040530](https://github.com/MetaMask/metamask-extension/pull/26870/commits/104053089b0ce889c735acd496055c50fe08acf2)